components/Mission.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+const Mission = ({MissionImage, MissionTitle,MissionText}) => {
```

Comment:
change the variable name to camelCase with starting with `lowercase` convention is to use classes and component names starting from uppwercase.

Reply:
Forgot that😅